### PR TITLE
locksmith v0.3.1

### PR DESCRIPTION
**Pull request by jonboulle**
/cc @crawford

### CODE REVIEW

**Comment by jonboulle** [comment]
Actually hold off, I am going to add a changelog.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+### v0.3.1
+
+v0.3.1 is the first release with a changelog :-)
+
+There are also a number of new features in this release, including [groups](README.md#groups), an [`off` strategy](README.md#configuration), and [reboot windows](README.md#reboot-windows).
+
+Changes since v0.3.0:
+- addition of a "groups" feature (#70), facilitating partitioned co-ordinating of reboots
```
Comment (steveeJ) [inline CHANGELOG.md]: maybe list the new features in level 2 under one level 1 item
- new features
  * groups
  * "off" strategy
  * reboot windows

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+- addition of an "off" strategy which will cause locksmith to perform no action and shut itself down (#79)
+- addition of reboot windows, allowing control over when reboots occur (#80)
+- greater verbosity of error messages in the case of unlocking failures (#82)
+- daemon now considers strategy when attempting to unlock, rather than just blindly checking the local etcd's activeness (#86)
+- updateengine client no longer attempts to close a dbus connection if the connecion failed (#83)
```
Comment (steveeJ) [inline CHANGELOG.md]: analog to features, maybe highlight bugfixes with an obvious level 1 keyword?

**Comment by jonboulle** [comment]
@steveeJ ptal

**Comment by steveeJ** [comment]
LGTM

**Pull request merged**